cmd/list.go

```diff
@@ -78,12 +78,11 @@ dapr list -k --all-namespaces
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		if kubernetesMode {
-			print.WarningStatusEvent(os.Stdout, "In future releases, this command will only query the \"default\" namespace by default. Please use the -n (namespace) flag, for specific namespace, or -A (all-namespaces) flag for all namespaces.")
+			print.WarningStatusEvent(os.Stdout, "In future releases, this command will only query the \"default\" namespace by default. Please use the --namespace flag, for specific namespace, or -A (--all-namespaces) flag for all namespaces.")
```

Comment:
Should the short-hand notation for `--all-namespaces` be `-a` instead of `-A`? Lowercasing will be consistent.

Reply:
Since we are using this warning string at multiple places, we can also consider moving this to a common file `messages.go` or something, refer it here.

Reply:
in this PR we tried to follow k8s short hands as much as possible ... and `-A` is fine ... we do have other short hands in the cli which are capitals.